plugins/lookup/github_app_access_token.py

```diff
@@ -47,6 +47,14 @@
       - How long the token should last for in seconds.
     default: 600
     type: int
+  github_api_url:
```

Comment:
In the `github_*` modules, this parameter is called `github_url` (github_deploy_key, github_webhook_info, github_webhook) or `api_url` (github_key, github_repo). Once there was a PR which tried to unify this (#85), and it chose `github_url`. Let's use `github_url` here as well instead of adding a third name :)